src/internal/document_api/mod.rs

```diff
@@ -1700,6 +1700,7 @@ mod tests {
         let aes_value = AesEncryptedValue::try_from(&[42u8; 32][..])?;
         let uid = UserId::unsafe_from_string("userid".into());
         let gid = GroupId::unsafe_from_string("groupid".into());
+        let did = TryFrom::try_from(1).unwrap();
```

Comment:
You should also be able to do 
`let did: DeviceId = 1.try_into()?;`

Reply:
I second this.

Reply:
Changed. I tried to find other places where I did this and changed those too.